coding-concepts.js

```diff
@@ -18,38 +18,38 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 const cohort = "LEARN 2023"
 // console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: my answer was 9
+// b) Verify and explain: .length reads the amount of characters from the string and gives you the answer
```

Comment:
The output is 10! There are 10 characters in the string, the space counts as a character.